templates/nginx_internal.conf

```diff
@@ -10,7 +10,7 @@ http {
     resolver {{ nginx_internal_resolver }} valid=5s;
     # set the real_ip when from docker internal ranges. Ensuring our internal nginx
     # container can always see the correct ips in the logs
-    set_real_ip_from 172.0.0.0/8;
+    set_real_ip_from 172.16.0.0/12;
```

Comment:
Please add another line for `10.88.0.0/16`, which Podman uses by default. Alternatively, add the default ranges as noted in your comment.

Reply:
updated to all rfc 1918 private ranges now